.github/workflows/2fa-check.yaml

```diff
@@ -0,0 +1,12 @@
+on: [push]
```

Comment:
this should be changed to run as a scheduled job (cron job) to run weekly or some other cadence